config/samples/config.yaml

```diff
@@ -1,26 +1,69 @@
 clusterName: "example-cluster"
 timeIntervalSec: 60
 repoCredentials:
-  "key1":
-    HTTPCredential:
-      Username: "exampleUser"
-      Password:
-        name: "http-creds"
-        key: "password"
-  "key2":
-    SSHCredential:
-      SSHKey:
-        name: "ssh-creds"
-        key: "sshKey"
-  "key3":
-    TLS:
-      insecureSkipVerify: false
-      CACertSecret:
-        name: "ca-cert-secret"
-        key: "ca.crt"
-      certSecret:
-        name: "cert-secret"
-        key: "tls.crt"
-      KeySecret:
-        name: "key-secret"
-        key: "tls.key"
+  - url: "github.com/rustyTest"
+    credential:
+      httpCredential:
+        username: "exampleUser"
+        password:
+          name: "http-creds"
+          key: "password"
+      sshCredential:
+        sshKey:
+          name: "ssh-creds"
+          key: "sshKey"
+      tls:
+        insecureSkipVerify: true
+        caCertSecret:
+          name: "ca-cert-secret"
+          key: "ca.crt"
+        certSecret:
+          name: "cert-secret"
+          key: "tls.crt"
+        keySecret:
+          name: "key-secret"
+          key: "tls.key"
```

Comment:
As far as I understand, I believe we would either use `insecureSkipVerify` or define all of those other fields, but not both.

Reply:
Ok ,Acknowledged .I am addressing that in another PR

Reply:
I know it's challenging having multiple PRs at the same time, but it will make sure we don't lose sight of things if we do things within the PR where the issue is caught. For example, your other PR may not end up getting merged in the end for whatever reason, so it's possible the issue will get forgotten.